deploy/kessel-relations.yaml

```diff
@@ -34,7 +34,7 @@ objects:
             token: "${PRESHARED}" # token takes precedence over tokenFile
             tokenFile: "${PRESHARED_FILE:.secrets/local-spicedb-secret}"
             schemaFile: "${SCHEMA_FILE:deploy/schema.zed}"
-            fullyConsistent: true
+            fullyConsistent: false
```

Comment:
I presume we can we pin stage/prod to an older commit, so notifications would still have fullyConsistent = true until they are ready to cut over? They will need an updated java client as well. (Will create a jira for updating clients.)

Reply:
this is for ephemeral and i would advise to keep the old value i.e "true" for now and once clients are ready, we can flip it over ? WDYT?

Reply:
Yup correct for stage/prod we manage it via app-interface configs. I will revert the change for ephemeral till we are ready to flip over.